### PR TITLE
Stack creature tokens of same power and toughness

### PR DESCRIPTION
**Pull request by grimreap124**
Stacks creature tokens but will split when the following is different; power/toughness, counters, abilities, summoning sickness.
image

Things to note, if abilities are given in different orders the tokens will not stack.

### CODE REVIEW

File: Mage.Client/src/main/java/mage/client/game/BattlefieldPanel.java [hunk 1]
```diff
@@ -43,11 +43,15 @@ public class BattlefieldPanel extends javax.swing.JLayeredPane {
 
     private static final int GAME_REDRAW_TIMEOUT_MS = 300; // timeout before game goes to redraw on scrollbars change
 
-    // WARNING, permanents contains top level PANELS (cards), use getMainPanel for real MagePermanent (permanents)
+    // WARNING, permanents contains top level PANELS (cards), use getMainPanel for
+    // real MagePermanent (permanents)
```
[the remark this answers is not in the document]
Comment (JayDi85) [reply Mage.Client/src/main/java/mage/client/game/BattlefieldPanel.java:47]: I think it related to word wrap settings, must be values like 120 or disabled at all. Example for redhat java extension: https://stackoverflow.com/a/76923883
org.eclipse.jdt.core.formatter.lineSplit=120
org.eclipse.jdt.core.formatter.comment.line_length=120

[the remark this answers is not in the document]
Comment (grimreap124) [reply Mage.Client/src/main/java/mage/client/game/BattlefieldPanel.java:47]: Thanks I'll update these tomorrow. I never write java so I left everything as default

[the remark this answers is not in the document]
Comment (grimreap124) [reply Mage.Client/src/main/java/mage/client/game/BattlefieldPanel.java]: There is just a check box for formatting comments so I unchecked it. If you can send me some eclipse java formatting that you guys use I'll be happy to use them so I'm not formatting everything I work on.

https://github.com/redhat-developer/vscode-java/wiki/Formatter-settings

[the remark this answers is not in the document]
Comment (JayDi85) [reply Mage.Client/src/main/java/mage/client/game/BattlefieldPanel.java]: You can try that eclipse compatible xml-config (I exported it from default IntelliJ IDEA):
[intellij-idea-default-code-style.zip](https://github.com/magefree/mage/files/15371345/intellij-idea-default-code-style.zip)

**Comment by grimreap124** [comment]
@JayDi85 I don't know if the Intellij formatter is the same that everyone else is using.  When saving different files to test it updates the formatting to something I think is less readable.

**Comment by grimreap124** [comment]
> You can use formatter in new files like [1247cb2](https://github.com/magefree/mage/commit/1247cb204bc6f1ea5f0b008d395573bf0a28264a) — if it generate many changes then something wrong in settings.
> 
> Also PlayerImpl.java and HumanPlayer.java can be a good target for experiment.
> 
> P.S. It’s ok to use different format styles in PR - just make sure it applied to real changes, not to other code/files.

Ok I might just stick to the default formatter for VS code as I think the huge if statements are hard to read, and I'll turn off the comment formatting

**Pull request merged**